tutorials/power_distribution/api.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "libs/gpio/api.h"
+
+extern gpio_t DASHBOARD_POWER;
+extern gpio_t SERVICE_SECTION_POWER;
+extern gpio_t THROTTLE_POWER;
+
+/*
+ * Initialize power distribution device pins
+ */
+void init_devices(void);
+
+/*
+ * Power a device
```

Comment:
typically you'd also notate parameters for functions that require them but nbd here - you get the point